<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TextEditor::Internal::ColorSchemeEdit</class>
 <widget class="QWidget" name="TextEditor::Internal::ColorSchemeEdit">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>462</width>
    <height>389</height>
   </rect>
  </property>
  <layout class="QGridLayout" name="gridLayout" columnstretch="1,0,0,0">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <property name="verticalSpacing">
    <number>0</number>
   </property>
   <item row="5" column="1" colspan="3">
    <widget class="QLabel" name="relativeForegroundHeadline">
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Relative Foreground</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLabel" name="foregroundLabel">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Foreground:</string>
     </property>
     <property name="buddy">
      <cstring>foregroundToolButton</cstring>
     </property>
    </widget>
   </item>
   <item row="10" column="1" colspan="3">
    <widget class="QWidget" name="relativeForegroundSpacer3" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>18</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>18</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="9" column="1">
    <widget class="QLabel" name="foregroundLightnessLabel">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Lightness:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QToolButton" name="foregroundToolButton">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="26" column="1" colspan="3">
    <spacer name="spacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>200</width>
       <height>0</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="3" column="3">
    <widget class="QToolButton" name="eraseBackgroundToolButton">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="toolTip">
      <string>Erase background.</string>
     </property>
     <property name="text">
      <string>x</string>
     </property>
     <property name="arrowType">
      <enum>Qt::LeftArrow</enum>
     </property>
    </widget>
   </item>
   <item row="14" column="1" colspan="3">
    <widget class="QWidget" name="relativeBackgroundSpacer2" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>6</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>6</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="7" column="2" colspan="2">
    <widget class="QDoubleSpinBox" name="foregroundSaturationSpinBox">
     <property name="minimum">
      <double>-1.000000000000000</double>
     </property>
     <property name="maximum">
      <double>1.000000000000000</double>
     </property>
     <property name="singleStep">
      <double>0.050000000000000</double>
     </property>
    </widget>
   </item>
   <item row="15" column="1">
    <widget class="QLabel" name="backgroundLightnessLabel">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Lightness:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="3">
    <widget class="QToolButton" name="eraseForegroundToolButton">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="toolTip">
      <string>Erase foreground.</string>
     </property>
     <property name="text">
      <string>x</string>
     </property>
     <property name="arrowType">
      <enum>Qt::LeftArrow</enum>
     </property>
    </widget>
   </item>
   <item row="13" column="1">
    <widget class="QLabel" name="backgroundSaturationLabel">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Saturation:</string>
     </property>
    </widget>
   </item>
   <item row="19" column="2" colspan="2">
    <widget class="QCheckBox" name="italicCheckBox">
     <property name="text">
      <string>Italic</string>
     </property>
    </widget>
   </item>
   <item row="9" column="2" colspan="2">
    <widget class="QDoubleSpinBox" name="foregroundLightnessSpinBox">
     <property name="minimum">
      <double>-1.000000000000000</double>
     </property>
     <property name="maximum">
      <double>1.000000000000000</double>
     </property>
     <property name="singleStep">
      <double>0.050000000000000</double>
     </property>
    </widget>
   </item>
   <item row="6" column="1" colspan="3">
    <widget class="QWidget" name="relativeForegroundSpacer1" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>6</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>6</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="13" column="2" colspan="2">
    <widget class="QDoubleSpinBox" name="backgroundSaturationSpinBox">
     <property name="minimum">
      <double>-1.000000000000000</double>
     </property>
     <property name="maximum">
      <double>1.000000000000000</double>
     </property>
     <property name="singleStep">
      <double>0.050000000000000</double>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QLabel" name="backgroundLabel">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Background:</string>
     </property>
     <property name="buddy">
      <cstring>backgroundToolButton</cstring>
     </property>
    </widget>
   </item>
   <item row="4" column="1" colspan="3">
    <widget class="QWidget" name="backgroundSpacer" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>18</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>18</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="0" column="0" rowspan="27">
    <widget class="QListView" name="itemList">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>1</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::ExtendedSelection</enum>
     </property>
     <property name="uniformItemSizes">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="17" column="1">
    <widget class="QLabel" name="fontHeadline">
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Font</string>
     </property>
    </widget>
   </item>
   <item row="7" column="1">
    <widget class="QLabel" name="foregroundSaturationLabel">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Saturation:</string>
     </property>
    </widget>
   </item>
   <item row="19" column="1">
    <widget class="QCheckBox" name="boldCheckBox">
     <property name="text">
      <string>Bold</string>
     </property>
    </widget>
   </item>
   <item row="18" column="1" colspan="3">
    <widget class="QWidget" name="fontSpacer1" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>6</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>6</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="12" column="1" colspan="3">
    <widget class="QWidget" name="relativeBackgroundSpacer1" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>6</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>6</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="16" column="1" colspan="3">
    <widget class="QWidget" name="relativeBackgroundSpacer3" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>18</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>18</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="24" column="1" colspan="3">
    <widget class="QWidget" name="underlineSpacer2" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>6</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>6</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="22" column="1" colspan="3">
    <widget class="QWidget" name="underlineSpacer1" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>6</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>6</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="23" column="2">
    <widget class="QToolButton" name="underlineColorToolButton">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="21" column="1">
    <widget class="QLabel" name="underlineHeadline">
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Underline</string>
     </property>
    </widget>
   </item>
   <item row="23" column="1">
    <widget class="QLabel" name="underlineLabel">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Color:</string>
     </property>
     <property name="buddy">
      <cstring>backgroundToolButton</cstring>
     </property>
    </widget>
   </item>
   <item row="11" column="1" colspan="3">
    <widget class="QLabel" name="relativeBackgroundHeadline">
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Relative Background</string>
     </property>
    </widget>
   </item>
   <item row="20" column="1" colspan="3">
    <widget class="QWidget" name="fontSpacer2" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>18</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>18</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="15" column="2" colspan="2">
    <widget class="QDoubleSpinBox" name="backgroundLightnessSpinBox">
     <property name="minimum">
      <double>-1.000000000000000</double>
     </property>
     <property name="maximum">
      <double>1.000000000000000</double>
     </property>
     <property name="singleStep">
      <double>0.050000000000000</double>
     </property>
    </widget>
   </item>
   <item row="3" column="2">
    <widget class="QToolButton" name="backgroundToolButton">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="23" column="3">
    <widget class="QToolButton" name="eraseUnderlineColorToolButton">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="toolTip">
      <string>Erase background.</string>
     </property>
     <property name="text">
      <string>x</string>
     </property>
     <property name="arrowType">
      <enum>Qt::LeftArrow</enum>
     </property>
    </widget>
   </item>
   <item row="25" column="1" colspan="3">
    <widget class="QComboBox" name="underlineComboBox"/>
   </item>
   <item row="8" column="1" colspan="3">
    <widget class="QWidget" name="relativeForegroundSpacer2" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>6</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>6</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="2" column="1" colspan="3">
    <widget class="QWidget" name="foregroundSpacer" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>6</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>6</height>
      </size>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>itemList</tabstop>
  <tabstop>foregroundToolButton</tabstop>
  <tabstop>eraseForegroundToolButton</tabstop>
  <tabstop>backgroundToolButton</tabstop>
  <tabstop>eraseBackgroundToolButton</tabstop>
  <tabstop>boldCheckBox</tabstop>
  <tabstop>italicCheckBox</tabstop>
  <tabstop>underlineColorToolButton</tabstop>
  <tabstop>eraseUnderlineColorToolButton</tabstop>
  <tabstop>underlineComboBox</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
